README.md

```diff
@@ -28,12 +28,13 @@ Any Spark environment should work, but for reference this was tested on Databric
 
 ### Dependencies
 
-pip install apache-sedona shapely
+pip install apache-sedona shapely pyproj
 
 Following packages versions were used:
 ```
 apache-sedona = "1.5.1"
 shapely = "2.0.2"
+pyproj = "3.6.1"
```

Comment:
maybe omit the specific version of the package over here, as these versions can quickly become out of sync with actual version being used in the `requirements.txt`:  the `apache-sedona` version defined in the `requirements.txt` is already different from the one here in the `README.md`

Reply:
makes sense, i replaced it with link to requirements instead